rosdep/base.yaml

```diff
@@ -3426,6 +3426,9 @@ portaudio:
   debian: [libportaudio-dev]
   fedora: [portaudio-devel]
   ubuntu: [libportaudio-dev]
+portaudio19:
```

Comment:
This should have the `-dev` extension. Our policy is to by default copy the debian name.